csvqb/csvqb/models/cube/qb/components/unit.py

```diff
@@ -24,28 +25,45 @@
 
 @dataclass
 class QbUnit(QbDataStructureDefinition, ABC):
-    @abstractmethod
-    def unit_multiplier(self) -> Optional[int]:
-        pass
+    pass
 
 
 @dataclass
 class ExistingQbUnit(QbUnit):
     unit_uri: str
-    unit_multiplier: Optional[int] = field(default=None, repr=False)
 
     _unit_uri_validator = validate_uri("unit_uri")
 
+    def __eq__(self, other):
+        return isinstance(other, ExistingQbUnit) and other.unit_uri == self.unit_uri
+
+    def __hash__(self):
+        return self.unit_uri.__hash__()
+
 
 @dataclass
 class NewQbUnit(QbUnit, UriIdentifiable, ArbitraryRdf):
     label: str
     description: Optional[str] = field(default=None, repr=False)
-    unit_multiplier: Optional[int] = field(default=None, repr=False)
-    parent_unit_uri: Optional[str] = field(default=None, repr=False)
     source_uri: Optional[str] = field(default=None, repr=False)
     uri_safe_identifier_override: Optional[str] = field(default=None, repr=False)
     arbitrary_rdf: List[TripleFragmentBase] = field(default_factory=list, repr=False)
+    base_unit: Optional[QbUnit] = field(default=None, repr=False)
+    """The unit that this new unit is based on."""
+    base_unit_scaling_factor: Optional[float] = field(default=None, repr=False)
```

Comment:
Should the default `base_unit_scaling_factor` be default to `1` instead of `None`? What difference would this have?